infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/RestApiRequest.java

```diff
@@ -31,7 +32,12 @@
   public <T> T getRequestBody() throws JsonProcessingException {
     DeserializableTypeDefinition<T> bodySchema =
         (DeserializableTypeDefinition<T>) metadata.getRequestBodyType();
-    return JsonUtil.parse(context.body(), bodySchema);
+
+    final String body = context.body();
+    if (body.equals("")) {
+      throw new MissingRequestBodyException();
+    }
+    return JsonUtil.parse(body, bodySchema);
```

Comment:
Personally I think I would have just passed the body to `JsonUtil.parse` but then thrown an error if it turned out to be `null`.  Currently we're likely to get `NullPointerException` if I post just whitespace or other interesting variations of null (e.g. the string `"null"` is valid JSON that results in `null`).

Reply:
makes sense. done.